Packs/TeamCymru/README.md

```diff
@@ -1,6 +1,22 @@
 Note: Support for this Pack was moved to Partner starting July 29, 2024. In case of any issues arise, please contact the Partner directly at <support@cymru.com>.

```

Comment:
```suggestion
Note: Support for this pack was moved to Partner starting July 29, 2024. In case any issues arise, contact the Partner directly at <support@cymru.com>.

```